### PR TITLE
Hotfix log files

### LINKED ISSUE #737: Using --logFile command with the logFile as the address doesn't work.
#### System information

- **OS & Version**: Windows/Linux/OSX
- **Go version:** 
- **razor-go version:** 

#### Backtrace

````
[backtrace]
````

### PR DESCRIPTION
**Pull request by Yashk767**
# Description

Moved logFiles from `./razor` to `./razor/logFiles` directory

Fixes #737

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -741,7 +741,7 @@ In the above example for the collection `ethCollectionMean`, new custom job havi
 
 ### Logs
 
-User can pass a separate flag --logFile followed with any name for log file along with command. The logs will be stored in ```.razor``` directory.
+User can pass a separate flag --logFile followed with any name for log file along with command. The logs will be stored in ```.razor/logFiles``` directory.
```
Comment (ashish10677) [inline README.md:744]: Change the name to just logs.
`./razor/logs`

**Review by ashish10677 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Yashk767) [reply README.md:744]: okay

**Review by ashish10677 — APPROVED** [review]

**Comment by ashish10677** [comment]
@Yashk767 tests are failing

**Comment by Yashk767** [comment]
> @Yashk767 tests are failing

fixed.

**Pull request merged**